graviti/portex/ptype.py

```diff
@@ -123,6 +124,31 @@ def check(arg: TypingAny) -> Sequence[TypingAny]:
         return arg
 
 
+class Enum(ParameterType):
+    """Parameter type for Portex enum values."""
+
+    @staticmethod
+    def check(arg: TypingAny) -> List[str]:
+        """Check and transfer the parameter type.
+
+        Arguments:
+            arg: The argument which needs to be checked.
+
+        Raises:
+            ValueError: When the value of the input argument does not fit the pattern.
+
+        Returns:
+            A list of enum values created by the input argument.
+
+        """
+        for value in arg:
```

Comment:
shall we check whether the arg is List? Or shall we check the value is string first?